installer/installer.py

```diff
@@ -51,18 +51,21 @@
 DEFAULT_DAEMON_POLL_INTERVAL = "300s"
 DEFAULT_UNSAFE_SKIP_BACKUP = "true"
 DEFAULT_DAEMON_RESTART_DELAY = "120s"
-
+DAEMON_DOWNLOAD_MUST_HAVE_CHECKSUM="true"
+DAEMON_SHUTDOWN_GRACE="30s"
 
 ###############################################################
 ###     			Systemd configuration      				###
 ###############################################################
 STANDALONE_SERVICE_TEMPLATE = f"https://raw.githubusercontent.com/cheqd/cheqd-node/{DEFAULT_DEBUG_BRANCH}/installer/templates/cheqd-noded.service"
 COSMOVISOR_SERVICE_TEMPLATE = f"https://raw.githubusercontent.com/cheqd/cheqd-node/{DEFAULT_DEBUG_BRANCH}/installer/templates/cheqd-cosmovisor.service"
+COSMOVISOR_CONFIG_TEMPLATE = f"https://raw.githubusercontent.com/cheqd/cheqd-node/{DEFAULT_DEBUG_BRANCH}/installer/templates/cosmovisor-config.toml"
 JOURNAL_TEMPLATE=f"https://raw.githubusercontent.com/cheqd/cheqd-node/{DEFAULT_DEBUG_BRANCH}/installer/templates/journald.conf"
 DEFAULT_STANDALONE_SERVICE_NAME = 'cheqd-noded'
 DEFAULT_COSMOVISOR_SERVICE_NAME = 'cheqd-cosmovisor'
 DEFAULT_STANDALONE_SERVICE_FILE_PATH = f"/lib/systemd/system/{DEFAULT_STANDALONE_SERVICE_NAME}.service"
 DEFAULT_COSMOVISOR_SERVICE_FILE_PATH = f"/lib/systemd/system/{DEFAULT_COSMOVISOR_SERVICE_NAME}.service"
+DEFAULT_COSMOVISOR_CONFIG_FILE_PATH = f"{DEFAULT_CHEQD_HOME_DIR}/.cheqdnode/cosmovisor/config.toml"
```

Comment:
Possible document this as well.

Reply:
Upgrading guide should suffice.

Reply:
https://docs.cheqd.io/node/getting-started/setup-and-configure/cosmovisor-configuration